app/assets/stylesheets/base/reset.scss

```diff
@@ -4,7 +4,13 @@
 [class^='crayons']::after,
 [class^='crayons'] *,
 [class^='crayons'] *::before,
-[class^='crayons'] *::after {
+[class^='crayons'] *::after,
+[class^='c-'],
+[class^='c-']::before,
+[class^='c-']::after,
+[class^='c-'] *,
+[class^='c-'] *::before,
+[class^='c-'] *::after {
   box-sizing: border-box;
 }
```

Comment:
Not nice. But we still need to define it in reset... I promise to clean this up soon :D